ui/src/components/group/AddGroup.js

```diff
@@ -168,11 +181,26 @@ export default class AddGroup extends React.Component {
             return;
         }
 
+        if (groupName.includes('/')) {
```

Comment:
let's use this https://github.com/yahoo/athenz/blob/master/core/zms/src/main/rdl/Names.tdl#L60 regex pattern and perform a check against it for the group name